components/var_dumper.rst

```diff
@@ -257,6 +272,18 @@ Example::
     {
         use VarDumperTestTrait;
 
+        protected function setUpVarDumper(array $casters, int $flags = null): void
```

Comment:
Actually, this should not be implemented, but called instead:
- either in `setUp`, so all test cases in the class will use the same casters & flags
- either specifically at the beginning of each test case methods (here `testWithDumpEquals`) when these test cases each have their specific needs.